config/default/manager_default_images.yaml

```diff
@@ -14,7 +14,8 @@ spec:
         - name: RELATED_IMAGE_OS_CONTAINER_IMAGE_URL_DEFAULT
           value: quay.io/podified-antelope-centos9/edpm-hardened-uefi:current-podified
         - name: RELATED_IMAGE_AGENT_IMAGE_URL_DEFAULT
-          value: quay.io/openstack-k8s-operators/openstack-baremetal-operator-agent:latest
+          # FIXME: REMOVE before merging
+          value: quay.io/andrewbays/openstack-baremetal-operator-agent:v0.0.1
```

Comment:
should be removed?

Reply:
Yes, but if we remove it in this PR it will break the jobs for it.  So ideally we would merge this, let the new agent builder push the new image, and then do a follow-up to change this back.